andela/apollo-ah-backend
# Feat/164198185/create articles

#### What does this PR do?
ensure users can create articles

#### Description of Task to be completed?
- create article models
- create associations between users model and articles model
- create a route for posting articles
- ensure articles are being created
- ensure authors can't have similar titles for different articles

#### How should this be manually tested?
- clone/download the branch
- install node.js 
- run npm install to install dependencies
- create .env from .env.example
- create .env.test from .env.example and configure
- create a database on Postgres and add the database name in .env
- create a database on postgres for tests and add databse name in .env.test
- run npm run migrate
- run npm run start: dev
- send a POST request to ```/api/v1/articles/``` to post an article
- user should get an object of created article in response

#### Any background context you want to provide?
- the information from the section above assumes a user account has beeen created before post requests are made to articles route
- the information from the section above assumes users are knowledagble about programming

#### What are the relevant pivotal tracker stories?
- #164198185
